react/src/components/awards/SprkAward.test.js

```diff
@@ -24,11 +24,10 @@ describe('SprkAward:', () => {
       },
     ];
     const wrapper = mount(<SprkAward images={images} />);
-    expect(
-      wrapper.find(
-        'div.sprk-o-Stack.sprk-o-CenteredColumn.sprk-o-Stack--medium',
-      ).length,
-    ).toBe(1);
+    expect(wrapper.find('div').first().hasClass('sprk-o-Stack')).toBe(true);
+    expect(wrapper.find('div').first().hasClass('sprk-o-Stack--medium')).toBe(
+      true,
+    );
```

Comment:
We should add something to this test to make it more explicit and make sure that no unnecessary classes are being added. A length check or find something to be false.